WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -14518,6 +14520,9 @@
 			dependencies = (
 			);
 			name = UITestsFoundation;
+			packageProductDependencies = (
+				3FC2C33C26C4CF0A00C6D98F /* XCUITestHelpers */,
```

Comment:
XCUITestHelpers is linked only to UITestsFoundation, yet it's available in WordPressUITests as you can see below.

I'm surprised by this. 🤔 IRRC, my past experience with dependencies and frameworks was that I'd have to link libraries to each consumer target, regardless of whether one of the frameworks already linked had the new library as a dependency. But maybe I'm wrong...

Anyway, it work on my machine and on CI, too 🤷‍♂️